src/main/java/co/novu/common/base/NovuConfig.java

```diff
@@ -1,16 +1,34 @@
 package co.novu.common.base;
 
+import org.apache.maven.model.Model;
+import org.apache.maven.model.io.xpp3.MavenXpp3Reader;
+
+import java.io.FileReader;
+
 import lombok.Getter;
 import lombok.Setter;
 
 @Getter
 @Setter
 public class NovuConfig {
 
+    private String apiKey;
+    private String baseUrl = "https://api.novu.co/v1/";
+    private String sdkName;
+    private String sdkVersion;
```

Comment:
Having these fields here will give the users the liberty to provide the name and version which shouldn't be. Please move this implementation into the RestHandler.java class.

By doing that, you don't need these fields, just have a function that returns the version, the name can be hardcoded, it should be **Java**